### PR TITLE
Add a list & definitions of some types parties involved in online advertising.

### PR DESCRIPTION
**Pull request by michaelkleber**
Brief discussion of the roles of publishers, advertisers, publisher ad platforms, DSPs, and ad servers.  By no means is this a complete list, but it covers the most important parties that touch the browser, so should be helpful for our discussions.

### CODE REVIEW

File: UseCasesofOnlineAdvertising.md [hunk 1]
```diff
@@ -3,15 +3,49 @@
...
 
 This document describes event sequences supporting modern online advertising. Many details are elided. There is a significant amount of tutorial literature available which can be used to fill in the details and omissions that are necessarily present in this summary document.
 
-## Ad Call with Single Market
+## Dramatis Personae
+
+The web advertising ecosystem is a collaboration among many parties.  Common designations like "third-party" can often be ambiguous when a single pageload involves interactions with servers run by many organizations with different roles, needs, and capabilities.  So we begin with a list of types of parties and what roles they play.
 
+* **Publisher:**
+  Publishers produce web pages that users visit.  Most of the time, you can think of the publisher as the first party (though aggregation and syndication sites can add a little confusion).
```
Comment (vsekhar) [inline UseCasesofOnlineAdvertising.md]: Is it worth pointing out the user's perspective? I.e. first-party means the user has often taken some action to navigate to that page, would likely say that is where they "are" if asked, and is the URL/origin/brand they see at the top of their browser. These user-side properties are unique to publishers.

Comment (michaelkleber) [reply UseCasesofOnlineAdvertising.md]: Feel free to suggest a change to this PR, of course!  But seems to me that when a user is visiting an advertiser site, it has all the properties that you mention as well.

**Comment by boz6336** [comment]
good ,good lucky my dear

**Comment by boz6336** [comment]
go to moon

File: UseCasesofOnlineAdvertising.md (hunk 1 above)
Comment (Mustafsjamal) [reply UseCasesofOnlineAdvertising.md:19]: Tnxx.  For.  All

**Pull request merged**